VideoAPI/VideoApi.AcceptanceTests/Steps/ConferenceSteps.cs

```diff
@@ -251,6 +265,32 @@ public void ThenTheJudgeInHearingResponseShouldBeRetrieved()
             expectedHearing.Username.Should().Be(judge.Username);
         }
 
+        private void CreateMultipleConferences(bool addDuplicateFirstNames)
+        {
+            _context.Test.ConferenceDetailsResponses = new List<ConferenceDetailsResponse>();
+            string judge1 = "Automation_" + Name.First() + RandomNumber.Next();
+            string judge2 = "Automation_" + Name.First() + RandomNumber.Next();
+            
+            for (int i = 0; i < 2; i++)
```

Comment:
you have hard coded this so it will only ever make 2 conferences. Can you reuse the existing method for creating a single hearing multiple times and pass in the name you want to duplicate? Better that than duplicating the code.

Reply:
Well spotted